Disable cleanup on remote work dir (#5742)
nextflow-io/nextflow: docs/reference/config.md

: If `true`, on a successful completion of a run all files in *work* directory are automatically deleted.

  :::{warning}
  The use of the `cleanup` option will prevent the use of the *resume* feature on subsequent executions of that pipeline run.

>   :::

Reviewer: Double `:::` crept in here. Also it's indented, which feels unusual?

Reviewer 2: These are two warnings, one after the other so the double `:::` is expected. The indentation is also normal as it's listed within the reference list.

Reviewer: Ok fab, thanks for checking!

  :::{warning}

>   The `cleanup` option is not supported for remote work directory such as Amazon S3, Google Cloud Storage, and Azure Blob Storage.

Reviewer 2: ```suggestion
  The `cleanup` option is not supported for remote work directories, such as Amazon S3, Google Cloud Storage, and Azure Blob Storage.
```

: If `true`, on a successful completion of a run all files in *work* directory are automatically deleted.

  :::{warning}

>   The use of the `cleanup` option will prevent the use of the *resume* feature on subsequent executions of that pipeline run.
>   Also, this capability is not supported by remote work directory such as AWS S3, Google Storage and Azure Blob container.

Reviewer 3: ```suggestion
  The use of the `cleanup` option will prevent the use of the *resume* feature on subsequent executions of that pipeline run.
  :::
  :::{warning}
  The `cleanup` option is not supported for remote work directory such as Amazon S3, Google Cloud Storage, and Azure Blob Storage.
```
